benchmarks/kubectl-mtb/internal/kubectl-mtb/run.go

```diff
@@ -130,6 +140,9 @@ func validateFlags(cmd *cobra.Command) error {
 		return fmt.Errorf("tenant namespace must be set via --namespace or -n")
 	}
 
+	benchmarkRunOptions.OtherNamespace, _ = cmd.Flags().GetString("other-namespace")
+	benchmarkRunOptions.OtherTenant, _ = cmd.Flags().GetString("other-tenant-admin")
```

Comment:
Maybe we can add the arguments in a list itself like ` -n test  test2 --as bob-k8s-access bob2-k8s-access`?

Reply:
That's a great idea! @phoenixking25 what do you think?

Reply:
Looks like the Cobra CLI package allows this:

```
cmd --slice a --slice b --slice c,d,e
```

https://github.com/spf13/cobra/issues/661